README.md

```diff
@@ -1,4 +1,4 @@
-# Picover
+# Picover [![Dev branch verification](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml) [![Pull Request verification](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml)
```

Comment:
how does exactly the pull request verification works ? 

It is showing that the last PR is passing ? or what ?

Reply:
It shows the last CI status. When the latest build will failed it will be red and with information failed.

For example, the screen which I show in the description showed the latest CI status when pull request and develop CI worked correctly.

Also if you tap on any badges you will be moved to every CI build for a specific type (pull requests, develop).